{
  "Header": {
    "Original": "CVE-2022-38143 (A heap out-of-bounds write vulnerability exists in the way OpenImageIO ...)",
    "Line": 57525,
    "ID": "CVE-2022-38143",
    "Description": "(A heap out-of-bounds write vulnerability exists in the way OpenImageIO ...)"
  },
  "Annotations": [
    {
      "Original": "- openimageio 2.4.7.1+dfsg-2 (bug #1027143)",
      "Line": 57526,
      "Type": "package",
      "Package": "openimageio",
      "Kind": "fixed",
      "Version": "2.4.7.1+dfsg-2",
      "BugNo": 1027143
    },
    {
      "Original": "[bullseye] - openimageio \u003cnot-affected\u003e (The vulnerable code was introduced later)",
      "Line": 57527,
      "Type": "package",
      "Release": "bullseye",
      "Package": "openimageio",
      "Kind": "not-affected",
      "Description": "The vulnerable code was introduced later"
    },
    {
      "Original": "[buster] - openimageio \u003cnot-affected\u003e (The vulnerable code was introduced later)",
      "Line": 57528,
      "Type": "package",
      "Release": "buster",
      "Package": "openimageio",
      "Kind": "not-affected",
      "Description": "The vulnerable code was introduced later"
    },
    {
      "Original": "NOTE: https://talosintelligence.com/vulnerability_reports/TALOS-2022-1630",
      "Line": 57529,
      "Type": "NOTE",
      "Description": "https://talosintelligence.com/vulnerability_reports/TALOS-2022-1630"
    },
    {
      "Original": "NOTE: https://github.com/OpenImageIO/oiio/pull/3620",
      "Line": 57530,
      "Type": "NOTE",
      "Description": "https://github.com/OpenImageIO/oiio/pull/3620"
    }
  ]
}